main.c

```diff
@@ -32,10 +35,46 @@ int main(int argc, char *argv[]) {
   // validates if a is able to be multiplied by b.
   if (matrix_mult_valid(a, b))
   {
-    Matrix_t t = matrix_transpose(a);
-    matrix_print(t);
+	Matrix_t c = matrix_create(a.rows,b.cols);
+    
+	//Matrix_t t = matrix_transpose(a);
+    //matrix_print(t);
+
+    //Matrix_t c = matrix_mult(a, b);
```

Comment:
Instead of commenting `matrix_mult`, I would define a `matrix_mult_openmp` within matrix module and would move all the `openMP` logic into this new function. Our `main` function should not be aware of how matrix multiplication works because matrix module is in charge of doing it.